File: C#/Pila/Carro.cs
//Nombre clase
public class Carro {

    //Atributos
    string marca, placa;

    //Constructor y getter, setter
    public Carro(string marca, string placa) {
        this.marca = marca;
        this.placa = placa;
    }

    public string getMarca() {
        return this.marca;
    }

   
    public string getPlaca() {
        return this.placa;
    }

    public void setMarca(string marca) {
        this.marca = marca;
    }
   
    public void setPlaca(string placa) {
        this.placa = placa;
    }

    public string mostrar() {
        return "*****************"+"\nMarca: " + marca + "\nPlaca: " 
        + placa+"\n*****************";
    }
}

File: JAVA/Pila/Carro.java
package JAVA.Pila;
//Nombre de la clase
public class Carro {

    //Atributos
    private String marca;
    private String placa;

    //Método constructor y getter, setter
    public Carro(String marca, String placa) {

        this.marca = marca;
        this.placa = placa;
    }

    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }
   

    public String getPlaca() {
        return placa;
    }

    public void setPlaca(String placa) {
        this.placa = placa;
    }

    public String toString() {
        return "*****************"+"\nMarca: " + marca + "\nPlaca: " 
        + placa+"\n*****************";
    }
}
